plaidml/bridge/tensorflow/tests/resnext_test.cc

```diff
@@ -0,0 +1,3473 @@
+// Tests that show HLO Module conversion to PlaidML Program.
+
+#include <gtest/gtest.h>
+
+#include <fstream>
+
+#include "flatbuffers/flatbuffers.h"
+#include "tensorflow/compiler/xla/service/hlo_parser.h"
+#include "tensorflow/compiler/xla/tests/test_utils.h"
+#include "tensorflow/compiler/xla/tests/verified_hlo_module.h"
+
+#include "plaidml/bridge/tensorflow/tests/archive_generated.h"
+#include "plaidml/bridge/tensorflow/tests/codegen_test.h"
+
+using plaidml::edsl::MultiBuffer;
+namespace zoo = plaidml::zoo;
+
+static MultiBuffer convertBuffer(const zoo::DataUnion& data) {
```

Comment:
Would be great if we could refactor this so that it's in the `codegen_test.cc`, so that it's not duplicated between multiple networks.

Reply:
I agree, I'll refactor this once I have ResNext working.